activesupport/lib/active_support/inflector/methods.rb

```diff
@@ -152,8 +156,9 @@ def humanize(lower_case_and_underscored_word, options = {})
     #   titleize('x-men: the last stand')    # => "X Men: The Last Stand"
```

Comment:
this is a good example to demo the ignore_dashes option on 
`#   titleize('x-men: the last stand', ignore_dashes: true)    # => "X-Men: The Last Stand"`